### PR TITLE
first: Nice work!

### PR DESCRIPTION
**Pull request by thebylito**
- Added some ES6 synthesizers.
- When constructing, a hash file is generated, so that a user who already has the file in his browser's cache loads the new file.
- Changed the readme file.

### CODE REVIEW

File: src/components/ReachMe.js [hunk 1]
```diff
@@ -4,27 +4,28 @@ import { connect } from "react-redux";
 import "./ReachMe.scss";
 
 const mapStateToProps = state => {
+  const { user, login } = state;
```
Comment (eveningkid) [inline src/components/ReachMe.js]: Could even be simplified to:
```js
const mapStateToProps = ({ user, login }) => { ... }
```

File: src/components/inspect/PersonalPhraseInspector.js [hunk 2]
```diff
@@ -6,9 +6,9 @@ import {editSession} from '../../actions'
 import './PersonalPhraseInspector.scss'
 
 const mapStateToProps = (state) => {
+  const { user, session } = state
```
Comment (eveningkid) [inline src/components/inspect/PersonalPhraseInspector.js]: Same here, can be put into `state` instead, right?

**Review by eveningkid — CHANGES_REQUESTED** [review]
Thank you very much for taking time to work on this. It's sort of an old project, I'm so glad you found it and was willing to help. 

Please, update the two small details I mentioned and we'll be good to merge. Thanks! 😃

File: src/components/inspect/PersonalPhraseInspector.js (hunk 2 above)
Comment (thebylito) [reply src/components/inspect/PersonalPhraseInspector.js]: yeah :)

**Review by eveningkid — APPROVED** [review]

**Pull request merged**